app/src/main/java/com/framgia/englishconversation/data/model/TimelineModel.java

```diff
@@ -227,6 +238,20 @@ public int getViewType() {
         return getMedias() != null ? getMedias().size() : 0;
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
WARNING: [OverrideBothEqualsAndHashcode] Ensure you override both equals() and hashCode()
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/basic.html#OverrideBothEqualsAndHashcode